### PR TITLE
Mensuration symbols Phase 1

### PR DESCRIPTION
**Pull request by seemantk**
@curran, this is ready for your review and merge -- this is phase 1 of the mensuration symbols (instead of font).  This is only the single symbol cases.  Compound symbols are just shown as text for now. 

mensurationsymbols

### CODE REVIEW

**Comment by curran** [comment]
@seemantk Awesome! Thanks for sharing this.

**Comment by seemantk** [comment]
@curran, sorry for the delay

**Comment by seemantk** [comment]
@curran, the current status is that the SVG symbol is shown instead of the text.

The next step is to size and position the SVGs appropriately for each tick.  The symbols are upside down, it looks like.

jos1913-mensyms

**Comment by seemantk** [comment]
Also, the symbols may actually be upside down. :/

**Comment by seemantk** [comment]
@curran, this is ready for your review and merge -- this is phase 1 of the mensuration symbols (instead of font).  This is only the single symbol cases.  Compound symbols are just shown as text for now. 

mensurationsymbols

File: index.html [hunk 1]
```diff
@@ -30,7 +30,70 @@
         <div id="main" class="notes col-md-8 col-lg-8">
           <div class="row">
             <div id="notesbook" class="well">
-              <svg></svg>
+              <svg>
+                <defs>
+                <symbol id="uniE910" viewBox="1 -266 531 531">
```
Comment (curran) [inline index.html:35]: Nice approach, loving how this plays out with the axis.

File: sympress/js/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+var glyphids = d3.range(2320, 2331).concat(d3.range(2342, 2346))
+			.map(function(g) { return "e" + g.toString(16); })
```
Comment (curran) [inline sympress/js/index.js:2]: One nitpicky thing here, we're using spaces throughout the project, but this file uses tabs.

**Review by curran — APPROVED** [review]
Looks great! Works very well. Excellent work.

**Pull request merged**